### ISSUE TITLE
Issue with SavedData and FeedOption after.

### ISSUE THREAD

**Issue by vikigenius** [halcyonnouveau/roux#39]
Following this comment: https://github.com/halcyonnouveau/roux/issues/36#issuecomment-1005303236

I am trying to iterate through my saved items. But using the after option just keeps giving me the items

```rs
        let saved1 = account.saved(None).await.unwrap();
        let last_child_id1 = match &saved1.data.children.last().unwrap().data {
            SavedData::Comment(comments_data) => comments_data.id.as_ref().unwrap(),
            SavedData::Submission(submissions_data) => &submissions_data.id,
        };
        dbg!("Child Last 1: {}", last_child_id1);
        let after_options = FeedOption::new().after(&saved1.data.after.unwrap());
        let saved2 = account.saved(Some(after_options)).await.unwrap();
        let last_child_id2 = match &saved2.data.children.last().unwrap().data {
            SavedData::Comment(comments_data) => comments_data.id.as_ref().unwrap(),
            SavedData::Submission(submissions_data) => &submissions_data.id,
        };
        dbg!("Child Last 2: {}", last_child_id2);
        assert_ne!(last_child_id1, last_child_id2);
```

The assertion fails. I also notice that it returns the same `after` value. I know for a fact that I have a lot of saved items, and I don't see any scenario where the same `after` value should be returned.

**Comment by beanpuppy** [comment]
Hi, sorry about all these problems you've been encountering, you're in the areas that I haven't touched very much. 

Anyway, there was an issue with how the urls were being built which has been fixed in this commit: 86b981ebc40f926d7d591f9650a1a26e3be61601. If you could test the commit and confirm it works that would be great.

**Comment by vikigenius** [comment]
@beanpuppy It's working fine now.

> Hi, sorry about all these problems you've been encountering, you're in the areas that I haven't touched very much.

Happy to help, I was planning on creating a new library anyway since yours doesn't support OAuth authentication, but it's too much hassle and your library is very convenient so I am happy with it for now.
